test/export/InstanceExporter.test.ts

```diff
@@ -1043,16 +1043,18 @@ describe('InstanceExporter', () => {
       });
     });
 
-    it('should fix an inline reference while resolving the Instance being referred to', () => {
+    it('should fix a reference to a contained resource using a relative reference', () => {
```

Comment:
A question to make sure I understand correctly: in this test case, `contained` gets a reference added to it, and it has the shortened `#id` form. This happens regardless of whether or not the instance being referenced is inline or not. But when a reference to an instance (inline or not) gets put into some element other than `contained`, should it always get the full `type/id` reference?

Reply:
Yes so `TestOrganization` is added to the `contained` array, and then the `managingOrganization` is set to a Reference to `TestOrganization`. Based on the documentation here: https://www.hl7.org/fhir/references.html#canonical-fragments and here: http://www.hl7.org/fhir/domainresource.html#invs, my understanding is that a resource in the `contained` array should always be referenced with `#id`. And yes, my understanding is that the `containd` array is special, these special reference rules do not apply in other contexts.